research-software-directory/RSD-as-a-service
# Cumulative fixes

Closes #219  
Closes #221  
Closes #351  
Closes #361   

Changes proposed in this pull request:

*   add accessibility tab
*   add icons to add menu and user menu
*   fixed error on empty secondary type for datacite mentions
*   fixed keyboard (tab) navigation on home page and sign in
*   the logo height on software page is defined to min 3rem (3x16px) and max 5rem (5x16px) which should address logo scaling

![](https://user-images.githubusercontent.com/4195550/176618552-c620274e-1fef-420c-8386-bbd5af4b1a0c.gif)

*   fix UI

![](https://user-images.githubusercontent.com/4195550/176618716-bdd0d83d-2d2f-4a9c-a523-57597012ee53.gif)

How to test:

*   run `make start`
*   navigate with your tab key to see the border when outlined

PR Checklist:
